genesyscloud/workforcemanagement_businessunits/resource_genesyscloud_workforcemanagement_businessunits_schema.go

```diff
@@ -0,0 +1,224 @@
+package workforcemanagement_businessunits
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+/*
+resource_genesycloud_workforcemanagement_businessunits_schema.go holds four functions within it:
+
+1.  The registration code that registers the Datasource, Resource and Exporter for the package.
+2.  The resource schema definitions for the workforcemanagement_businessunits resource.
+3.  The datasource schema definitions for the workforcemanagement_businessunits datasource.
+4.  The resource exporter configuration for the workforcemanagement_businessunits exporter.
+*/
+const resourceName = "genesyscloud_workforcemanagement_businessunits"
```

Comment:
Set `resourceName` to `ResourceName` so it can be referenced by other resource packages.